eng/common/scripts/add-pullrequest-reviewers.ps1

```diff
@@ -0,0 +1,95 @@
+param(
+    [Parameter(Mandatory = $true)]
+    $RepoOwner,
+
+    [Parameter(Mandatory = $true)]
+    $RepoName,
+
+    [Parameter(Mandatory = $false)]
+    $GitHubUsers = "",
+
+    [Parameter(Mandatory = $false)]
+    $GitHubTeams = "",
+
+    [Parameter(Mandatory = $true)]
+    $PRNumber,
+  
+    [Parameter(Mandatory = $true)]
+    $AuthToken
+)
+
+function CombineArrays($ExistingIds, $NewIds) {
+  if ($ExistingIds -and $ExistingIds.Length > 0) {
+    $results = $ExistingIds.Clone()
+  }
+  else {
+    $results = @()
+  }
+
+  foreach ($id in $NewIds) {
+    if (-not $results.Contains($id)) {
+      $results += $id
+    }
+  }
+
+  return ,$results
```

Comment:
LOL... I know you mentioned you did this for a reason about one or the other list being null but I think you can accomplish what you want by just doing:

```
$results = @()
$results += $ExistingIds
foreach ($id in $NewIds) {
    if (-not $results.Contains($id)) {
      $results += $id
    }
  }
return $results
```

Reply:
If results is empty, it resolves to Null on the other side. :/ Still need the unary.